etc/tests/test_integration.py

```diff
@@ -163,6 +163,7 @@ def test_vault():
 
     # confirm that this works with the CANDIG_S3_TOKEN too:
     headers["X-Vault-Token"] = ENV["VAULT_S3_TOKEN"]
+    print(f"vault token is {headers['X-Vault-Token']}")
```

Comment:
This can probably be removed...

Reply:
Resolved by [639c87b](https://github.com/CanDIG/CanDIGv2/pull/248/commits/639c87b8b1255f57ab21b2fa1b28c05d70b20a50)